sw/snRuntime/src/start.S

```diff
@@ -44,6 +44,45 @@ snrt.crt0.init_bss:
     blt       t0, t1, 1b
 2:
 
+snrt.crt0.init_registers:
+    # Clear FP registers
+    csrr    t0, misa
+    andi    t0, t0, (1 << 3) # 3: D - single precision float extension
```

Comment:
What about F-only cores? I think we should also use `f` instructions here, since they will also clear the register.

We could also set `mtvec` to the `3f` label. In case the core doesn't have the available instructions it should except to that address with an illegal instruction.

Reply:
Absolutely correct but I wouldn't do it with an exception. Resolved in 5d61898